template/{{.input_root_dir}}/.github/workflows/{{.input_project_name}}-run-tests-fs.yml.tmpl

```diff
@@ -1,65 +0,0 @@
-name: Feature and Training Integration Tests for {{template `project_name` .}}
```

Comment:
Yeah I didn't update the tar.gz yet. Was planning to do that once the PR was in a more final state to avoid having to generate over and over again. Once the workflow is deployed though this will be automatic so that will be nice

Reply:
Updated the tar.gz to include all the latest changes so should be good

Reply:
Jk looks like the workflow ran woohoo! We're good